rails/app/views/shared/_footer.html.haml

```diff
@@ -17,6 +17,6 @@
         = mail_to(APP_CONFIG[:help_email] || APP_CONFIG[:default_admin_user][:email], "Send us an email", :subject => "#{APP_CONFIG[:site_name]} question", :encode => "javascript")
         %br
         Version:
-        = ENV['CC_PORTAL_VERSION']
+        = ENV['CC_PORTAL_IMAGE_VERSION'].present? ? ENV['CC_PORTAL_IMAGE_VERSION'] : ENV['CC_PORTAL_VERSION']
```

Comment:
I find this confusing when looking at what is happening in the DockerFile. Where else should I be looking for these ENV params?

Reply:
In our normal deployment process CC_PORTAL_VERSION is set by CloudFormation here:

https://github.com/concord-consortium/cloud-formation/blob/7ecad89105ffc011559807fe6a9059e0289ea80b/portal-ecs.yml#L478-L486

That sets it based on the docker image tag that a developer sets with the `PortalDockerImage` param. When building the images directly on docker hub there isn't a way to embed a version directly into the image.  (at least there wasn't before).  So we had to determine the image based on an environment variable set when the container is being run. 
That is error prone since a mis configuration can set the version to be something completely different from what the image actually is. 

In CodeBuild we can use docker build arguments to pass the version into into the building of the image, which then embeds this version in the image itself.

Because our code needs to handle both deployment models the line above first checks for the embedded `CC_PORTAL_IMAGE_VERSION` and if it doesn't find it then it uses the non embedded `CC_PORTAL_VERSION`.

Hopefully at some point soon, we can drop the docker hub builds, and use CodeBuild for the production builds as well. Then we can simplify this logic.